abixen-platform-modules/src/main/resources/liquibase/import/data/sample-database-connection.csv

```diff
@@ -0,0 +1,2 @@
+id;created_by_id;created_date;last_modified_by_id;last_modified_date;database_host;database_name;database_port;database_type;description;name;password;username
```

Comment:
Rename file to database-connection.csv - similarly to the core microservice imports.